static/app/views/starfish/views/queryParameters.tsx

```diff
@@ -0,0 +1,4 @@
+export enum QueryParameterNames {
+  CURSOR = 'spansCursor',
+  SORT = 'spansSort',
+}
```

Comment:
There's a ton more query params that we use in starfish, `project`, `transaction`, `statsPeriod`, `start`, `end`.
Not sure about adding the date ones in there (statsperiod, start, end), as we should probably just access that via pageFilter hook. But thoughts on adding the rest of the query params there in the future?

Reply:
🤔 good point! I agree that `pageFilter` is the right place to use the more general ones. I think `QueryParameterNames` is probably not great as a name or concept, because these are a bit of a hack to allow both a `cursor` and a `spansCursor` query param, so we can have separate cursors. IMO a better solution here would be to introduce something like query parameter namespacing, so all Starfish-specific parameters would be prefixed with `span.` or something like that. I'm going to follow up on this after V0!

Reply:
Sounds good!